IDE/STARCORE/testwolfcrypt_8254/LCF/mmu_attr.l3k

```diff
@@ -0,0 +1,128 @@
+// mmu_attr_cpp.l3k
+// *** Copyright Freescale Semiconductor= Inc= 2010 *** 

```

Comment:
I don't think we can handle this copyright.

Reply:
Copyright files removed.